{{cookiecutter.project_app}}/{{cookiecutter.project_app}}/settings/base.py

```diff
@@ -270,13 +270,13 @@
     "ADMINS",
     [
         (
-            "caktus-{{ cookiecutter.project_slug }}-team",
-            "{{ cookiecutter.project_slug }}-team@caktusgroup.com",
+            "caktus-{{ cookiecutter.project_app }}-team",
+            "{{ cookiecutter.project_app }}-team@caktusgroup.com",
         )
     ],
 )
 
 GENERAL_INQUIRY_EMAIL_RECIPIENTS = [e[1] for e in ADMINS]
 EVENT_SIGNUP_FORM_SUBMISSION_RECIPIENTS = GENERAL_INQUIRY_EMAIL_RECIPIENTS
```

Comment:
`GENERAL_INQUIRY_EMAIL_RECIPIENTS` and `EVENT_SIGNUP_FORM_SUBMISSION_RECIPIENTS` are settings that shouldn't really be in here